src/README.md

```diff
@@ -60,6 +60,8 @@ Margin: https://tailwindcss.com/docs/margin/
 
 Padding: https://tailwindcss.com/docs/padding/
 
+The introduction of this new feature means the **mb** property on `<Heading />` & `<Text />` are now deprecated.
```

Comment:
I think we won't need this given we already have the **migration guide** documenting this change ✍🏻 👍🏻

Same for the docs of `<Heading />` and `<Text />`.